internal/service/verify/data_source_verify_voice_phrase.go

```diff
@@ -0,0 +1,243 @@
+package verify
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/patrickcping/pingone-go-sdk-v2/pingone/model"
+	"github.com/patrickcping/pingone-go-sdk-v2/verify"
+	"github.com/pingidentity/terraform-provider-pingone/internal/framework"
+	"github.com/pingidentity/terraform-provider-pingone/internal/sdk"
+	validation "github.com/pingidentity/terraform-provider-pingone/internal/verify"
+)
+
+// Types
+type VoicePhraseDataSource struct {
+	client *verify.APIClient
+	region model.RegionMapping
+}
+
+type voicePhraseDataSourceModel struct {
+	Id            types.String `tfsdk:"id"`
+	EnvironmentId types.String `tfsdk:"environment_id"`
+	VoicePhraseId types.String `tfsdk:"voice_phrase_id"`
+	DisplayName   types.String `tfsdk:"name"`
```

Comment:
I went back and forth on this because `verify_policy` API uses 'name'.  My intent was to try consistent naming across the verify configurations, and because `verify_policy` was already released, I didn't want to change its configuration arguments.  

Admittedly in the API, `verify_voice_phrase` uses display name - so the APIs are even inconsistent.  I'll change to `display_name`.